web/viewer.js

```diff
@@ -1025,7 +1032,14 @@ var PDFViewerApplication = {
         console.warn('Warning: AcroForm/XFA is not supported');
         self.fallback(PDFJS.UNSUPPORTED_FEATURES.forms);
       }
-
+      
```

Comment:
Nit: unnecessary change (only spaces). Please remove this and also try to adjust your editor to remove such whitespace.